tutorial/index.jade

```diff
@@ -82,7 +82,7 @@
       **See Also:**
       * [FirstOrDefault Method](//msdn.microsoft.com/en-us/library/vstudio/bb918914.aspx)
```

Comment:
On the slide `Route the call to an agent` adjust the highlight to include the comment